### PR TITLE
Add submodule Berlin

### PR DESCRIPTION
**Pull request by gislars**
I just discovered this repository this week. So I gave it a try and made a submodule for Berlin. It was straight forward, thanks for your good work!
Since I run into timeouts I had to update composer.json and overpass-way.php. The docs for GuzzleHttp mentions when no timeout is given, it'll wait until it finishes. But after 60-80 seconds a exception got raised. Adding the timeout parameter solved this problem.

I tried to put the final map online, but first I have to figure out how. Since I don't want to put it at $webroot, instead at example.org/equalstreetnames-berlin, I have to figure out how it works with nginx or if I have to configure the build somehow.

On the other hand, Berlin only have few name:etymology:wikidata tags, not much to see yet. But I hope this map will change it.

### CODE REVIEW

File: composer.json [hunk 1]
```diff
@@ -20,5 +20,8 @@
             "@statistics"
         ],
         "wikidata": "php scripts/wikidata.php"
+    },
+    "config": {
+        "process-timeout": 800
```
Comment (jbelien) [inline composer.json:25]: I would prefer not to specify any global process timeout. I guess 800 would be enough for most city but also can be too little for some.
I usually run `composer run update-data --timeout=0 -- --city=brussels`.
I'll disable the timeout for the `update-data` script only.

File: scripts/overpass-way.php [hunk 2]
```diff
@@ -43,7 +43,8 @@ function get(string $city): string
     $client = new \GuzzleHttp\Client();
     $response = $client->request(
         'GET',
-        sprintf('https://overpass-api.de/api/interpreter?data=%s', urlencode($query))
+        sprintf('https://overpass-api.de/api/interpreter?data=%s', urlencode($query)),
+        ['timeout' => 400]
```
Comment (jbelien) [inline scripts/overpass-way.php:47]: This is not necessary !
The default Guzzle behavior is no timeout (see https://docs.guzzlephp.org/en/stable/request-options.html#timeout)
The timeout for the Overpass query should only be defined in the 2 overpass query files.

**Comment by jbelien** [comment]
> I tried to put the final map online, but first I have to figure out how. Since I don't want to put it at $webroot, instead at example.org/equalstreetnames-berlin, I have to figure out how it works with nginx or if I have to configure the build somehow.

I use [Netlify](https://www.netlify.com/) for the hosting. The free "package" is enough to run it !

1. You connect your Netlify site to this repository (once this PR is merged)
2. You set `npm run build:berlin` as build command
3. You set `dist/berlin` as publish directory
4. Enjoy the result ! 🤩

File: composer.json (hunk 1 above)
Comment (jbelien) [reply composer.json:25]: Done 😄  
https://github.com/openknowledgebe/equalstreetnames/blob/a2ea526d2983a5fb1e9cf3844ed17b6d0a110856/composer.json#L16

**Comment by gislars** [comment]
Thanks for your feedback. I removed the timeout settings and can confirm it is working locally. I wanted to stash the commits, but failed to do it.
Next task for me: figure out how to bring it online.

**Comment by jbelien** [comment]
Have a look at my comment from this morning : https://github.com/openknowledgebe/equalstreetnames/pull/56#issuecomment-657186128

Ping me if you need more help !

**Pull request merged**